share/completions/pzstd.fish

```diff
@@ -14,8 +14,13 @@ end
 
 complete -c pzstd -l ultra -d "Enable compression level beyond 19"
 
-for threads in (seq 1 (count (cat /proc/cpuinfo | string match -r "processor")))
-    complete -x -c pzstd -s p -l processes -a "$threads" -d "De/compress using $threads working threads"
+## If Python 3.4 or later installed, the number of physical cores is assigned to a variable.
+test -z (__fish_anypython); or set -l physical_cores (eval (__fish_anypython)" -c 'import os; os.cpu_count() is not None and print(os.cpu_count())' ^ /dev/null")
```

Comment:
1. There's no need for `eval` here, fish since 3.0 allows variables as commands:

```fish
set -l python (__fish_anypython)
set -q python[1] and set -l physical_cores ($python -c ...)
```

2. The `^` redirection is deprecated, use `2>` instead